

*
* Visual Studio Code Server
*
* Reminder: You may only use this software with Visual Studio family products,
* as described in the license https://aka.ms/vscode-remote/license
*

Extension host agent listening on 34977

[13:33:34] Extension host agent started.
[13:33:35] [127.0.0.1][4d1fdabf][ManagementConnection] New connection established.
[13:33:35] [127.0.0.1][bcc2984f][ExtensionHostConnection] New connection established.
[13:33:35] [127.0.0.1][bcc2984f][ExtensionHostConnection] <8195> Launched Extension Host Process.
[13:43:04] [127.0.0.1][4d1fdabf][ManagementConnection] The client has disconnected gracefully, so the connection will be disposed.
[13:43:04] [127.0.0.1][bcc2984f][ExtensionHostConnection] <8195> Extension Host Process exited with code: 0, signal: null.
Last EH closed, waiting before shutting down
[13:43:04] Last EH closed, waiting before shutting down
Got delay-shutdown request while in shutdown timeout, delaying
[13:45:17] Got delay-shutdown request while in shutdown timeout, delaying
Cancelling previous shutdown timeout
[13:45:17] Cancelling previous shutdown timeout
[13:45:17] [127.0.0.1][9a7d04bf][ManagementConnection] New connection established.
[13:45:17] [127.0.0.1][22715504][ExtensionHostConnection] New connection established.
[13:45:17] [127.0.0.1][22715504][ExtensionHostConnection] <8748> Launched Extension Host Process.
[IPC Library: Pty Host]  INFO Persistent process "6": Replaying 559 chars and 1 size events
New EH opened, aborting shutdown
[13:50:17] New EH opened, aborting shutdown
[14:08:38] [127.0.0.1][9a7d04bf][ManagementConnection] The client has disconnected gracefully, so the connection will be disposed.
[14:08:38] [127.0.0.1][22715504][ExtensionHostConnection] <8748> Extension Host Process exited with code: 0, signal: null.
Last EH closed, waiting before shutting down
[14:08:38] Last EH closed, waiting before shutting down
Got delay-shutdown request while in shutdown timeout, delaying
[14:09:02] Got delay-shutdown request while in shutdown timeout, delaying
Cancelling previous shutdown timeout
[14:09:02] Cancelling previous shutdown timeout
[14:09:03] [127.0.0.1][deecf7fa][ManagementConnection] New connection established.
[14:09:03] [127.0.0.1][60d8cd8e][ExtensionHostConnection] New connection established.
[14:09:03] [127.0.0.1][60d8cd8e][ExtensionHostConnection] <10050> Launched Extension Host Process.
[IPC Library: Pty Host]  INFO Persistent process "14": Replaying 694 chars and 1 size events
[IPC Library: Pty Host]  INFO Persistent process "15": Replaying 1908 chars and 1 size events
[IPC Library: Pty Host]  INFO Persistent process "16": Replaying 368 chars and 1 size events
[IPC Library: Pty Host]  INFO Persistent process "17": Replaying 503 chars and 1 size events
New EH opened, aborting shutdown
[14:14:02] New EH opened, aborting shutdown
[15:59:34] [File Watcher (parcel)] Unexpected error: std::bad_alloc (EUNKNOWN) (path: /home/austyn)
[15:59:34] [File Watcher (parcel)] restarting watcher after error: std::bad_alloc
[16:17:57] [127.0.0.1][86d410e2][ManagementConnection] New connection established.
[16:17:57] [127.0.0.1][4debfc9e][ExtensionHostConnection] New connection established.
[16:17:57] [127.0.0.1][4debfc9e][ExtensionHostConnection] <17134> Launched Extension Host Process.
rejected promise not handled within 1 second: Error: Could not find pty with id "26"
stack trace: Error: Could not find pty with id "26",    at l._throwIfNoPty (/home/austyn/.vscode-server/bin/899d46d82c4c95423fb7e10e68eba52050e30ba3/out/vs/platform/terminal/node/ptyHostMain.js:17:4919),    at l.updateTitle (/home/austyn/.vscode-server/bin/899d46d82c4c95423fb7e10e68eba52050e30ba3/out/vs/platform/terminal/node/ptyHostMain.js:17:1591),    at Object.call (/home/austyn/.vscode-server/bin/899d46d82c4c95423fb7e10e68eba52050e30ba3/out/vs/platform/terminal/node/ptyHostMain.js:13:8419),    at L.onPromise (/home/austyn/.vscode-server/bin/899d46d82c4c95423fb7e10e68eba52050e30ba3/out/vs/platform/terminal/node/ptyHostMain.js:12:4940),    at L.onRawMessage (/home/austyn/.vscode-server/bin/899d46d82c4c95423fb7e10e68eba52050e30ba3/out/vs/platform/terminal/node/ptyHostMain.js:12:4299),    at /home/austyn/.vscode-server/bin/899d46d82c4c95423fb7e10e68eba52050e30ba3/out/vs/platform/terminal/node/ptyHostMain.js:12:3552,    at R.fire (/home/austyn/.vscode-server/bin/899d46d82c4c95423fb7e10e68eba52050e30ba3/out/vs/platform/terminal/node/ptyHostMain.js:8:1712),    at process.Q (/home/austyn/.vscode-server/bin/899d46d82c4c95423fb7e10e68eba52050e30ba3/out/vs/platform/terminal/node/ptyHostMain.js:6:32910),    at process.emit (events.js:315:20),    at emit (internal/child_process.js:903:12),    at processTicksAndRejections (internal/process/task_queues.js:81:21)
[Error: Could not find pty with id "26",    at l._throwIfNoPty (/home/austyn/.vscode-server/bin/899d46d82c4c95423fb7e10e68eba52050e30ba3/out/vs/platform/terminal/node/ptyHostMain.js:17:4919),    at l.updateTitle (/home/austyn/.vscode-server/bin/899d46d82c4c95423fb7e10e68eba52050e30ba3/out/vs/platform/terminal/node/ptyHostMain.js:17:1591),    at Object.call (/home/austyn/.vscode-server/bin/899d46d82c4c95423fb7e10e68eba52050e30ba3/out/vs/platform/terminal/node/ptyHostMain.js:13:8419),    at L.onPromise (/home/austyn/.vscode-server/bin/899d46d82c4c95423fb7e10e68eba52050e30ba3/out/vs/platform/terminal/node/ptyHostMain.js:12:4940),    at L.onRawMessage (/home/austyn/.vscode-server/bin/899d46d82c4c95423fb7e10e68eba52050e30ba3/out/vs/platform/terminal/node/ptyHostMain.js:12:4299),    at /home/austyn/.vscode-server/bin/899d46d82c4c95423fb7e10e68eba52050e30ba3/out/vs/platform/terminal/node/ptyHostMain.js:12:3552,    at R.fire (/home/austyn/.vscode-server/bin/899d46d82c4c95423fb7e10e68eba52050e30ba3/out/vs/platform/terminal/node/ptyHostMain.js:8:1712),    at process.Q (/home/austyn/.vscode-server/bin/899d46d82c4c95423fb7e10e68eba52050e30ba3/out/vs/platform/terminal/node/ptyHostMain.js:6:32910),    at process.emit (events.js:315:20),    at emit (internal/child_process.js:903:12),    at processTicksAndRejections (internal/process/task_queues.js:81:21)]
rejected promise not handled within 1 second: Error: Could not find pty with id "26"
stack trace: Error: Could not find pty with id "26",    at l._throwIfNoPty (/home/austyn/.vscode-server/bin/899d46d82c4c95423fb7e10e68eba52050e30ba3/out/vs/platform/terminal/node/ptyHostMain.js:17:4919),    at l.updateIcon (/home/austyn/.vscode-server/bin/899d46d82c4c95423fb7e10e68eba52050e30ba3/out/vs/platform/terminal/node/ptyHostMain.js:17:1651),    at Object.call (/home/austyn/.vscode-server/bin/899d46d82c4c95423fb7e10e68eba52050e30ba3/out/vs/platform/terminal/node/ptyHostMain.js:13:8419),    at L.onPromise (/home/austyn/.vscode-server/bin/899d46d82c4c95423fb7e10e68eba52050e30ba3/out/vs/platform/terminal/node/ptyHostMain.js:12:4940),    at L.onRawMessage (/home/austyn/.vscode-server/bin/899d46d82c4c95423fb7e10e68eba52050e30ba3/out/vs/platform/terminal/node/ptyHostMain.js:12:4299),    at /home/austyn/.vscode-server/bin/899d46d82c4c95423fb7e10e68eba52050e30ba3/out/vs/platform/terminal/node/ptyHostMain.js:12:3552,    at R.fire (/home/austyn/.vscode-server/bin/899d46d82c4c95423fb7e10e68eba52050e30ba3/out/vs/platform/terminal/node/ptyHostMain.js:8:1712),    at process.Q (/home/austyn/.vscode-server/bin/899d46d82c4c95423fb7e10e68eba52050e30ba3/out/vs/platform/terminal/node/ptyHostMain.js:6:32910),    at process.emit (events.js:315:20),    at emit (internal/child_process.js:903:12),    at processTicksAndRejections (internal/process/task_queues.js:81:21)
[Error: Could not find pty with id "26",    at l._throwIfNoPty (/home/austyn/.vscode-server/bin/899d46d82c4c95423fb7e10e68eba52050e30ba3/out/vs/platform/terminal/node/ptyHostMain.js:17:4919),    at l.updateIcon (/home/austyn/.vscode-server/bin/899d46d82c4c95423fb7e10e68eba52050e30ba3/out/vs/platform/terminal/node/ptyHostMain.js:17:1651),    at Object.call (/home/austyn/.vscode-server/bin/899d46d82c4c95423fb7e10e68eba52050e30ba3/out/vs/platform/terminal/node/ptyHostMain.js:13:8419),    at L.onPromise (/home/austyn/.vscode-server/bin/899d46d82c4c95423fb7e10e68eba52050e30ba3/out/vs/platform/terminal/node/ptyHostMain.js:12:4940),    at L.onRawMessage (/home/austyn/.vscode-server/bin/899d46d82c4c95423fb7e10e68eba52050e30ba3/out/vs/platform/terminal/node/ptyHostMain.js:12:4299),    at /home/austyn/.vscode-server/bin/899d46d82c4c95423fb7e10e68eba52050e30ba3/out/vs/platform/terminal/node/ptyHostMain.js:12:3552,    at R.fire (/home/austyn/.vscode-server/bin/899d46d82c4c95423fb7e10e68eba52050e30ba3/out/vs/platform/terminal/node/ptyHostMain.js:8:1712),    at process.Q (/home/austyn/.vscode-server/bin/899d46d82c4c95423fb7e10e68eba52050e30ba3/out/vs/platform/terminal/node/ptyHostMain.js:6:32910),    at process.emit (events.js:315:20),    at emit (internal/child_process.js:903:12),    at processTicksAndRejections (internal/process/task_queues.js:81:21)]
[16:18:11] [127.0.0.1][86d410e2][ManagementConnection] The client has disconnected gracefully, so the connection will be disposed.
[16:18:11] [127.0.0.1][4debfc9e][ExtensionHostConnection] <17134> Extension Host Process exited with code: 0, signal: null.
[16:18:17] [127.0.0.1][cacb5eb1][ManagementConnection] New connection established.
[16:18:17] [127.0.0.1][7ac06c04][ExtensionHostConnection] New connection established.
[16:18:17] [127.0.0.1][7ac06c04][ExtensionHostConnection] <17288> Launched Extension Host Process.
[IPC Library: Pty Host]  INFO Persistent process "14": Replaying 750 chars and 1 size events
[IPC Library: Pty Host]  INFO Persistent process "15": Replaying 1954 chars and 1 size events
[IPC Library: Pty Host]  INFO Persistent process "16": Replaying 414 chars and 1 size events
[IPC Library: Pty Host]  INFO Persistent process "17": Replaying 549 chars and 1 size events
[IPC Library: Pty Host]  INFO Persistent process "22": Replaying 8619 chars and 1 size events
